### PR TITLE
new icon: vitejs (original, plain)

### LINKED ISSUE #1053: Icon request: Vite
### About the icon

Vite is an open source build system similar to Webpack used by newer versions of Vuejs.

### Links and sources

Vite's official website: https://vitejs.dev/
Github: https://github.com/vitejs/vite
The logo is available as SVG: https://vitejs.dev/logo.svg

### Additional information

Vite is created by Evan You, the same guy who created Vue.js.

### PR DESCRIPTION
**Pull request by kilianpaquier**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #1053**

This PR can be closed : https://github.com/devicons/devicon/pull/1252

## Link to prove your SVG is correct and up-to-date.

https://vitejs.dev/

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -8092,6 +8092,24 @@
         "color": "#68217A",
         "aliases": []
     },
+    {
+        "name": "vitejs",
+        "tags": [
+            "react",
```
Comment (Snailedlt) [inline devicon.json:8098]: vite.js isn't exclusive to react, so I think we should replace it with javascript, and also add the build tag
```suggestion
            "javascript",
            "build"
```

File: devicon.json [hunk 2]
```diff
@@ -8092,6 +8092,24 @@
         "color": "#68217A",
         "aliases": []
     },
+    {
+        "name": "vitejs",
```
Comment (Snailedlt) [inline devicon.json:8096]:
```suggestion
        "name": "vitejs",
        "altnames": [
            "vite.js",
            "Vite"
        ]
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Thanks for picking up this PR, looks good for the most part, just a few things missing

File: devicon.json (hunk 2 above)
Comment (kilianpaquier) [reply devicon.json:8096]: I could apply the suggestion, just, could you put back the closing square bracket ?

Comment (Snailedlt) [reply devicon.json:8096]: It's already there, just github's visuals that's acting weird for some reason.

This is what I added
```json
        "name": "vitejs",
        "altnames": [
            "vite.js",
            "Vite"
        ],
```

Comment (Snailedlt) [reply devicon.json:8096]: I had forgotten the comma though, so added that :)

Comment (kilianpaquier) [reply devicon.json:8096]: Ok, applying suggestions then :)

File: devicon.json [hunk 3]
```diff
@@ -8092,6 +8092,29 @@
...
+        "altnames": [
+            "vite.js",
+            "Vite"
+        ],
+        "tags": [
+            "javascript",
+            "build"
```
Comment (kilianpaquier) [inline devicon.json:8103]:
```suggestion
            "build",
```

**Review by Snailedlt — APPROVED** [review]
LGTM! ✔️ 

Beautiful 😘👌
Thank you for the quick fizes whcih made reviewing a breeze, and ofc thank you for the contribution! 💯

File: devicon.json [hunk 4]
```diff
@@ -8092,6 +8092,29 @@
...
+        "tags": [
+            "javascript",
+            "build",
+            "compiler", 
+            "esbuild"
+        ],
+        "versions": {
+            "svg": [
+                "original"
+            ],
```
Comment (lunatic-fox) [inline devicon.json:8110]:
```suggestion
            "svg": [
                "original",
                "plain"
            ],
```

**Review by lunatic-fox — CHANGES_REQUESTED** [review]

**Review by Snailedlt — DISMISSED** [review]

**Review by Snailedlt — APPROVED** [review]

**Pull request merged**